--- _examples/foo/foo.go	2022-11-14 10:42:56.000000000 +0900
+++ _examples/foo_transformed/foo.go	2022-11-14 10:55:42.000000000 +0900
@@ -2,20 +2,18 @@
 
 import (
 	"fmt"
-
-	"github.com/pkg/errors"
 )
 
 func Foo() error {
 	if err := Bar(); err != nil {
-		return errors.Wrap(err, "foo --")
+		return fmt.Errorf("foo -- %w", err)
 	}
 	return nil
 }
 
 func Bar() error {
 	if err := Boo(); err != nil {
-		return errors.Wrapf(err, "on %s%d", "bar", 0)
+		return fmt.Errorf("on %s%d -- %w", "bar", 0, err)
 	}
 	return nil
 }
@@ -25,7 +23,7 @@
 
 func Multiples() (bool, error) {
 	if err := Bar(); err != nil {
-		return false, errors.Wrap(err, "multiples")
+		return false, fmt.Errorf("multiples -- %w", err)
 	}
 	return true, nil
 }
@@ -39,7 +37,7 @@
 
 func WithSprintf() error {
 	if err := Bar(); err != nil {
-		return errors.Wrap(err, fmt.Sprintf("error message with sprintf %d", 0))
+		return fmt.Errorf(fmt.Sprintf("error message with sprintf %d", 0)+" -- %w", err)
 	}
 	return nil
 }
@@ -49,5 +47,5 @@
 		fmt.Println("hmm")
 		return err
 	}
-	return newXXXError(errors.Wrap(Boo(), "hmm"))
+	return newXXXError(fmt.Errorf("hmm -- %w", Boo()))
 }
