--- _examples/foo/is_test.go	2022-11-14 12:02:57.000000000 +0900
+++ _examples/foo_transformed/is_test.go	2022-11-14 12:14:00.000000000 +0900
@@ -4,7 +4,7 @@
 	"fmt"
 	"testing"
 
-	"github.com/pkg/errors"
+	"errors"
 )
 
 func TestIs(t *testing.T) {
@@ -12,7 +12,7 @@
 	if want, got := xxx, xxx; !errors.Is(got, want) {
 		t.Errorf("unexpected: %v != %v", want, got)
 	}
-	if want, got := xxx, xxx; !errors.Is(errors.Wrap(got, "hmm"), want) {
+	if want, got := xxx, xxx; !errors.Is(fmt.Errorf("hmm -- %w", got), want) {
 		t.Errorf("unexpected: %v != %v", want, got)
 	}
 
